lib/config/executor/network_interface.go

```diff
@@ -60,17 +60,26 @@ type ContainerNetwork struct {
 
 	// The network scope the IP belongs to.
 	// The IP address is the default gateway
-	Gateway net.IPNet `vic:"0.1" scope:"read-write" key:"gateway"`
+	Gateway net.IPNet `vic:"0.1" scope:"read-only" key:"gateway2"`
```

Comment:
This is a replacement of old gateway attribute. It might worth a version 0.2, or equal to 0.8 release